extensions/eclipse-che-theia-file-sync-tracker/README.md

```diff
@@ -0,0 +1,13 @@
+# theia-file-sync-tracker extension
+
+This extension tracks events of file synchronization progress during workspace startup.
+
+### How does the extension work?
+
+If workspace configured to use asynchronous storage extension will open websocket connection to special service and show progress in status bar. In case some error, error message will show in status bar after clicking will appear popup with error details. After successfully file synchronization status bar also will update with corresponding message, in this case message will disappear in 5 seconds.
```

Comment:
Just some spelling things, fyi.

> storage extension will open

storage **,** extension will open

> error message will show

error message will **be** shown

> in status bar after clicking

in status bar **,** after clicking

> after clicking will appear popup with error details

the popup with error details will be disappeared after mouse click

> file synchronization status bar also

file synchronization **,** status bar also

> also will update

also will **be** updated